mcs/class/System/System.Net/MacProxy.cs

```diff
@@ -326,15 +391,144 @@ public override string ToString ()
 		{
 			return Create (str);
 		}
+
+		// to be used when an API like CF*Get* returns a CFString
+		internal static string FetchString (IntPtr handle)
```

Comment:
That was called by:

               internal static string FetchString (IntPtr handle, bool releaseHandle)

which was used in one place.

But I think it'd be cleaner to just inline that release in the calling code.